editor/modes/visual-editor/block-list.js

```diff
@@ -71,7 +104,22 @@ class VisualEditorBlockList extends wp.element.Component {
 	}
 
 	onSelectionStart( uid ) {
+		const { pageYOffset } = window;
+
+		// Create a Y coödinate map to unique block IDs.
+		this.coordMap = reduce( this.refs, ( acc, node, blockUid ) => ( {
+			...acc,
+			[ pageYOffset + node.getBoundingClientRect().top ]: blockUid,
+		} ), {} );
+		// Cache an array of the Y coödrinates for use in `onPointerMove`.
```

Comment:
Found this: http://www.newyorker.com/culture/culture-desk/the-curse-of-the-diaeresis